packages/OverflowMenu/src/OverflowMenu.styles.js

```diff
@@ -10,3 +11,13 @@ export const Content = styled.div`
     ${stylers.lineHeight(-1)}
   }
 `;
+
+export const PopoverCard = styled(Popover.Card)(
+  ({ maxHeight }) => css`
+    max-height: ${maxHeight}px;
+    overflow: auto;
+    ::-webkit-scrollbar {
```

Comment:
Whats the UX spec for the scrollable list? I currently just have the scroll bar hidden.

Reply:
I think it's better to show it – it provides a visual affordance, a status of where in the list you currently are, and an indication of how large the list is / how much is not shown.  It should be flush with the edges of the `<Popover.Card>`, but I think that's what it will look like by default?